oap-server/oal-grammar/pom.xml

```diff
@@ -27,6 +27,10 @@
 
     <artifactId>oal-grammar</artifactId>
 
+    <properties>
+        <antlr.version>4.7.1</antlr.version>
+    </properties>
+
```

Comment:
Why this version is managed in specific project?

Reply:
> Why this version is managed in specific project?

This is plugin version, plugin dependency cannot be managed by bom